.github/CODEOWNERS

```diff
@@ -5,4 +5,4 @@
 ##############################################################
 
 # These owners will be the default owners for everything in the repo.
-* @genomic-medicine-sweden/gms-16s-devs
+* @genomic-medicine-sweden/taco-devs
```

Comment:
Seems to be an error being flagged here...

Reply:
@ryanjameskennedy Hmm, yea, this will be in a way "wrong" until we do the name change everywhere, but somehow seems better to change it once and for all?